commons/connector/fitbit/fitbit_food_log.avsc

```diff
@@ -0,0 +1,108 @@
+{
+  "namespace": "org.radarcns.connector.fitbit",
+  "type": "record",
+  "name": "FitbitFoodLog",
+  "doc": "A summary of the user's food log entry for a given day.",
+  "fields": [
+    {
+      "name": "time",
+      "type": "double",
+      "doc": "Device timestamp in UTC (s)."
+    },
+    {
+      "name": "timeReceived",
+      "type": "double",
+      "doc": "Time that the data was received from the Fitbit API (seconds since the Unix Epoch)."
+    },
+    {
+      "name": "isFavorite",
+      "type": "boolean",
+      "doc": "Whether the food is the favorite one."
+    },
+    {
+      "name": "logId",
+      "type": "long",
+      "doc": "The id of the logged food."
+    },
+    {
+      "name": "accessLevel",
+      "type": "string",
+      "doc": "The access level of the food."
+    },
+    {
+      "name": "amount",
```

Comment:
What is the unit of this?

Reply:
From the `unitName` of the food, so for example amount 1 could be 1 sandwich if unit is sandwich (see example response on the [api page](https://dev.fitbit.com/build/reference/web-api/nutrition/get-food-log/))